Functions.Templates/Templates/EdgeHubTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,19 @@
+{

```

Comment:
This can be removed

Reply:
Will remove. Can you share why it's not required? I see this file exists in other templates.